src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="bigBlock" 
+    method="post"
+    class="bigBar"
+    data-qa="big"
+>
```

Comment:
```suggestion
    <form 
      action="bigBlock" 
      method="post"
      class="bigBar"
      data-qa="big"
    >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/220853646-11569596-5314-41a7-bf6d-b617eae870f0.png)